bin/install

```diff
@@ -25,15 +44,22 @@ install_crc() {
         exit
     fi
 
-    download="https://mirror.openshift.com/pub/openshift-v4/clients/crc/${install_version}/crc-${os}-amd64.tar.xz"
-
     if [ ! -d "${install_path}/bin" ]; then
         mkdir -p "${install_path}/bin"
     fi
 
     echo "This is a large file to download: $(curl -sI $download | grep "Content-Length:" | sed 's/Content-Length: //g' | tr -d '[:space:]' | if [ "$os" = "macos" ]; then gnumfmt --to=iec; else numfmt --to=iec; fi)"
 
-    curl -fL# -N "${download}" | tar -x -J -C "${install_path}/bin" --strip-components=1
+    # CRC is only avail as an OSX pkg install after 1.27.0
+    if [[ "$os" == "macos" && $(semver_compare $install_version 1.26.0) == 1 ]]; then
+        download="https://mirror.openshift.com/pub/openshift-v4/clients/crc/${install_version}/crc-${os}-amd64.pkg"
+        curl -L# -N -o crc-download.pkg "${download}"
+        installer -pkg crc-download.pkg -target ${install_path}
```

Comment:
My testing is still on going (as pulling 3.26GB is not exactly fast), so there might be one or two minor changes to the `installer` line. Hopefully that will be resolved before any action is to be taken on this PR.